CODEOWNERS

```diff
@@ -5,13 +5,12 @@
 /python/ @github/codeql-python
 /ruby/ @github/codeql-ruby
 
-# Make @xcorail (GitHub Security Lab) a code owner for experimental queries so he gets pinged when we promote a query out of experimental
-/cpp/**/experimental/**/* @github/codeql-c-analysis @xcorail
-/csharp/**/experimental/**/* @github/codeql-csharp @xcorail
-/java/**/experimental/**/* @github/codeql-java @xcorail
-/javascript/**/experimental/**/* @github/codeql-javascript @xcorail
-/python/**/experimental/**/* @github/codeql-python @xcorail
-/ruby/**/experimental/**/* @github/codeql-ruby @xcorail
+/cpp/**/experimental/**/* @github/codeql-c-analysis
+/csharp/**/experimental/**/* @github/codeql-csharp
+/java/**/experimental/**/* @github/codeql-java
+/javascript/**/experimental/**/* @github/codeql-javascript
+/python/**/experimental/**/* @github/codeql-python
+/ruby/**/experimental/**/* @github/codeql-ruby
```

Comment:
```suggestion
```
These lines are implied by the block above, so they aren't needed

Reply:
That's true :facepalm: I certainly could have done a better job on the initial PR, thanks :+1: